scss/helpers/_color-bg.scss

```diff
@@ -0,0 +1,10 @@
+// stylelint-disable declaration-no-important, function-name-case
+
+// All-caps `RGBA()` function used because of this Sass bug: https://github.com/sass/node-sass/issues/2251
+@each $color, $value in $theme-colors {
+  $color-rgb: to-rgb($value);
+  .text-bg-#{$color} {
+    color: color-contrast($value) !important;
+    background-color: RGBA($color-rgb, var(--#{$prefix}bg-opacity, 1)) !important;
```

Comment:
Why is the RGBA function in uppercase ?

Reply:
There is an explanation in the file: `// All-caps `RGBA()` function used because of this Sass bug: https://github.com/sass/node-sass/issues/2251`